packages/victory-core/src/victory-theme/types.ts

```diff
@@ -194,7 +194,6 @@ export interface VictoryThemeDefinition {
     offsetY?: number;
     orientation?: OrientationTypes;
   } & VictoryCommonThemeProps;
-  label?: VictoryLabelStyleObject | VictoryLabelStyleObject[];
```

Comment:
I _believe_ this isn't actually getting read anywhere inside Victory? 

Though, in the future, it might be good to re-evaluate how to refactor the theme config definition so that label styles are set once at the top level of the config and not nested within multiple chart types in the theme config.